images/macos/provision/core/xcode.ps1

```diff
@@ -44,7 +44,11 @@ Invoke-XcodeRunFirstLaunch -Version $defaultXcode
 Write-Host "Configuring Xcode symlinks..."
 $xcodeVersions | ForEach-Object {
     Build-XcodeSymlinks -Version $_.link -Symlinks $_.symlinks
-    Build-ProvisionatorSymlink -Version $_.link
+
+    # Temporary solution to install Xcode 13 beta along with 13 stable
+    if ($_.link -ne "13.0_beta") {
```

Comment:
What do you think of making it more generalized? Like
```
    if ($_.skipsymlink -ne "true") {
        Build-ProvisionatorSymlink -Version $_.link
    }
```
and add `skipsymlink: true` for xcode_13_beta. It will help us if we have the same case in future